src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,69 @@
+package duke.command;
+
+import duke.date.Date;
```

Comment:
Maybe you could consider standardising the order of the import statements across your files to make it easier to browse through? Your code is really clean so I am picking on the minute details.

Reply:
Can I clarify what standard are you referring to? IntelliJ and CheckStyle seem to disagree, so I went with CheckStyle's import rules (if you are wondering).